sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md

```diff
@@ -5,12 +5,16 @@
 This version of the SDK defaults to the latest supported API version, which currently is `v3.2-preview.2`.
 
 ### Features Added
+- Added support for Custom Entities Recognition through the `begin_analyze_actions` API with the `RecognizeCustomEntitiesAction` and `RecognizeCustomEntitiesResult` types.
+- Added support for Custom Single Classification through the `begin_analyze_actions` API with the `SingleCategoryClassifyAction` and `SingleCategoryClassifyActionResult` types.
+- Added support for Custom Multi Classification through the `begin_analyze_actions` API with the `MultiCategoryClassifyAction` and `MultiCategoryClassifyActionResult` types.
 
 ### Breaking Changes
 
 ### Bugs Fixed
 
 ### Other Changes
+- Package requires [azure-core](https://pypi.org/project/azure-core/) version 1.16.0 or greater
```

Comment:
do we usually include this in he changelog? don't think it's wrong to include, just didn't know we were supposed to

Reply:
Good question. Before all the changelog header updates we had a section called "Dependency Updates" where this kind of info would normally go. There isn't really a place for it now which is why I put it under "Other Changes". Let me check the guidelines.